libpod/image/pull_test.go

```diff
@@ -243,12 +299,19 @@ func TestPullGoalFromImageReference(t *testing.T) {
 		if len(c.expected) == 0 {
 			assert.Error(t, err, c.srcName)
 		} else {
+			if c.mustFail {
```

Comment:
There’s already the `len(c.expected) == 0` check just above. Either just use it, or remove it and convert all its users to `mustFail`. Things like
```go
{"docker-archive:/dev/this-does-not-exist", nil, false, false, false}, // Input does not exist.
```
above are internally inconsistent.

Reply:
Fixed, thanks a lot!